sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/aio/_vendor.py

```diff
@@ -8,19 +8,30 @@
 from abc import ABC
 from typing import TYPE_CHECKING
 
-from ._configuration import ConversationAnalysisClientConfiguration
+from ._configuration import ConversationAuthoringClientConfiguration
```

Comment:
It looks like the sync code might have accidentally got copied into this file ;)

Reply:
How'd that get in there? (Spiderman 3 meme)